spoken/views.py

```diff
@@ -314,6 +314,7 @@ def watch_tutorial(request, foss, tutorial, lang):
     is_valid_user_for_tut = is_valid_user(request.user,foss,lang)
```

Comment:
this code is already there. remove this line also.

Reply:
Done